tests/test_layout.py

```diff
@@ -12,8 +12,8 @@ class TestLayout:
     @pytest.mark.nondestructive
     @pytest.mark.xfail("'mozilla.com' in config.getvalue('base_url')",
```

Comment:
Sorry, I suspect the documentation needs to be updated. A current limiting behavior with the test harness includes needing to specifically specify a `--base-url`.